client/view/QueryLink/queryLink.js

```diff
@@ -10,6 +12,10 @@ links.forEach(item => {
     } else {
       setFormValues({ query: item.innerText.trim() })
     }
+
     submitForm()
+
+    queryContainer.scrollTo({ top: 0 })
```

Comment:
`behavior: "smooth"`?

Reply:
Good idea. But it didn't work. :(

I'll check later why it didn't work for viewport (nothing happens either with the js method or when specifying the css property `scroll-behavior`)